movies.py

```diff
@@ -0,0 +1,77 @@
+"""Movie ticket process system"""
+
+MOVIE_PRICE = 5.0
+
+def enough_money(customer):
+    """Check if customer has adequate funds"""
+    return False if customer["cash"] < MOVIE_PRICE else True
+
+def purchase_ticket(movie, customer):
+    """Performs the steps required in purchasing a ticket and returns altered customer and movie"""
+    assert enough_money(customer)
+    customer = dispense_ticket(movie, charge(customer))
+    movie = remove_seat(movie)
+    return customer, movie
```

Comment:
i like this returning of the variables.

you could probably put them in the same order as you took them in though for consistency